### PR TITLE
Roboto font added

### PR DESCRIPTION
**Pull request by duskoo**

### CODE REVIEW

File: CoEpi/ui/healthquiz/QuestionCell.swift [hunk 1]
```diff
@@ -14,6 +15,17 @@ class QuestionCell: UITableViewCell {
             this.onChecked?(q, checked)
         }
         accessoryView = checkBox
+        guard let customFont = UIFont(name: "Roboto-Regular", size: 18) else {
```
Comment (i-schuetz) [inline CoEpi/ui/healthquiz/QuestionCell.swift]: We should put the fonts in a central place (`Fonts` or however you want to name it), to not have to repeat this code and cache them.

File: CoEpi/ui/home/HomeViewController.swift [hunk 2]
```diff
@@ -31,6 +37,8 @@ class HomeViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        //==Family: Roboto Font names: ["Roboto-Regular", "Roboto-Light", "Roboto-Bold"]
```
Comment (i-schuetz) [inline CoEpi/ui/home/HomeViewController.swift]: Delete comment if not needed.

**Comment by rase-rocks** [comment]
Here is an example of #27 in action. If this could be resolved it could reduce the amount of fiddling required for every commit

File: CoEpi/ui/healthquiz/QuestionCell.swift (hunk 1 above)
Comment (duskoo) [reply CoEpi/ui/healthquiz/QuestionCell.swift]: Added CoEpi/ui/resources/fonts/Fonts.swift

**Comment by duskoo** [comment]
Closing this PR. Please take a look at #29

**Pull request closed without merging**